src/pages/Rockets.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import RocketItem from '../components/rockets/elementRockets';
+import styles from '../components/rockets/Rockets.module.css';
 
 function Rockets() {
+  const rocketList = useSelector((state) => state.rocketReducer);
```

Comment:
Awesome using useSelector() Redux Hook to select the state slices and render lists of rockets in corresponding routes. 💯